src/PKSim.Core/Snapshots/Mappers/CurveMapper.cs

```diff
@@ -39,12 +43,24 @@ public CurveMapper(CurveOptionsMapper curveOptionsMapper, IDimensionFactory dime
          curve.CurveOptions.UpdateFrom(curveOptions);
 
          var yData = findCurveWithPath(snapshot.Y, simulationAnalysisContext.DataRepositories);
+         if (yData == null)
+         {
+            _logger.AddToLog(PKSimConstants.Error.CouldNotFindQuantityWithPath(snapshot.Y));
```

Comment:
That's the only thing that needs to be done when something should be logged. Grab the logger and write